### PR TITLE
Finish contact form project Pull request

### PR DESCRIPTION
**Pull request by HERMON-1995**
This project is about adding contact form section to the mobile first version project.

### CODE REVIEW

File: contact-data-received.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Data received from contact form</title>
+</head>
+<body>
+    <h2>Data received</h2>
+</body>
+</html>
```
Comment (leonardodiasb) [inline contact-data-received.html:12]: - [ ] This HTML file is not required and adds extra code to your project. Kindly remove it. You should use the Formspree service to use the form and there is no need to redirect the user to a different page.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+        </div>
+        </section>
+
+        <!--Form section-->
+        <section id="contact" class="form">
+          <div class="grid-2">
+
+           <div class="form-text"><h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3></div>
+
+           <form action="/contact-data-received.html" method="post" class="inputs">
```
Comment (leonardodiasb) [inline index.html:213]: - [ ] Form uses the action and method attribute to integrate with [formspree service](https://formspree.io/html). Kindly read the getting started section of the Formspree site and make sure you implement it correctly so the user can submit the form to you.
  - action value should be similar to https://formspree.io/f/{form_id}
  - method value equals POST

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @HERMON-1995, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Good UI design. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by didierganthier — APPROVED** [review]
Hi @HERMON-1995 ,

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀



## Highlights
- Linters passing ✔️ 
- Descriptive PR ✔️ 
- Correct GitHub flow ✔️ 

## Optional Suggestions 💡
- While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.

Cheers and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread and don't forget to tag me so I'll receive the notification.

------
As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s). 

Once again, congratulations on completing this project! 🎉

**Pull request merged**